src/components/globe-all-cities.tsx

```diff
@@ -27,7 +23,13 @@ const GlobeAllCities = component$<Props>(({ data, width = 390, height = 458 }) =
     };
   });
 
-  useVisibleTask$(() => {
+  useVisibleTask$(async () => {
+    const goeJson = await import('../utils/ne_110m_admin_0_countries.geojson.json');
+
+    const GlobeModule = await import('globe.gl');
+    const Globe = GlobeModule.default;
+    const THREE = await import('three');
```

Comment:
hey just out of curiosity, would there be any benefit of using Promise.all for these imports?

Reply:
Yeah it probably would but I had trouble with `globa.gl`, hence the `.default`. I was getting error that i didn't understand, so i hacked it :)